skimage/draw/_draw.pyx

```diff
@@ -392,12 +383,16 @@ def circle_perimeter(Py_ssize_t cy, Py_ssize_t cx, Py_ssize_t radius,
                 d = d + 2 * (y - x - 1)
                 y = y - 1
                 x = x + 1
+
+    ry = (np.array(rr, dtype=np.float) + cy)
+    rx = (np.array(cc, dtype=np.float) + cx)
+    ry = np.round(ry)
```

Comment:
Would it make sense to do ``ry = np.round(ry).astype(int)`` here to avoid allthe ``astype`` calls further down?  Same for ``rx``.